### ISSUE TITLE
 properties-maven-plugin adding current date to newly written file.

### ISSUE THREAD

**Issue by swadalkar** [mojohaus/properties-maven-plugin#71]
Hi,

my team is using properties-maven-plugin and we are using **write-project-properties** goal,
```
 <plugin>
				<groupId>org.codehaus.mojo</groupId>
				<artifactId>properties-maven-plugin</artifactId>
				<version>1.0-alpha-2</version>
				<executions>
					<execution>
						<phase>generate-resources</phase>
						<goals>
							<goal>write-project-properties</goal>
						</goals>
						<configuration>
							<outputFile>src/test/resources/reportportal.properties</outputFile>
						</configuration>
					</execution>
				</executions>
			</plugin> 
```
### _**_Properties file_**
#Properties
#Thu Feb 10 12:50:23 IST 2022      ### current date........
rp.endpoint=https\://www.google.com
rp.uuid=28f1d912-66d1-437b-8c12-0cf40330289d
java.version=1.8.0_271
maven.compiler.version=3.6.0_

it is writing down all the properties in given file path, but it also adding current date to that file, due to current date adding into file during git pull and push operation every time we need to revert back our changes from that file.
Is there any way to ignore adding current time into properties file.

**Comment by michael-o** [comment]
Not a plugin problem, but a Java problem. Please google. I have solved this for Maven Archiver years ago. Feel free to carve that out and provide a PR.

**Comment by hboutemy** [comment]
just upgrade the plugin to 1.1.0: this has been fixed, see #75 
(trying latest is usually a good practice before reporting :) )
